data/.gitignore

```diff
@@ -0,0 +1,5 @@
+*
```

Comment:
I know  '*' can be used in terminal to address different files with similar name. Like 'rm test*' can remove 'test1', 'test2', 'test3' etc. Why did you use it here? Or it doesn't matter because its just an example? Or because you wanted it to be hidden from me/public?

Reply:
I am talking about the asterisk symbol

Reply:
Good question. GitHub won't let you add empty folders that you might want if the directory structure is important for your code. So you can add a gitignore file and the folder will be added.
The folder is called data, so you can assume that's either an input or output folder. Either way, you wouldn't want to add the contents (or be able to because of GitHub size limitations) of that folder. But maybe you want it there just as a placeholder to show people how to setup their data.

Try adding a .gitignore file to one of your repos and just add the wildcard to it. See what happens. 

I'll make the changes tomorrow.